lib/index.js

```diff
@@ -113,7 +113,8 @@ exports.init = function(options, callback) {
     }
     
     var opt = {
-        depth: options.depth
+        depth: options.depth,
+        dev: options.include.indexOf('devDependencies') != -1,
```

Comment:
yes, I looked at the documentation for [`read-installed`](https://github.com/npm/read-installed), and it has a dev option. Not options for bundled or optional dependencies...